website/docs/tutorials/managing-access.md

```diff
@@ -44,6 +44,17 @@ A Resource's or Container's ACL is only accessible to your app if the following
 Access can be granted to individual [agents](../glossary#agent), to groups, or even to everyone.
 We currently only support reading what access has been granted to individual agents specifically.
 
+### Fetching access information
```

Comment:
OK, with this, I think I see where you're coming from: it feels weird to just fetch the metadata, only to have it wrapped in an object with a single property.

That said, if we want to be able to represent that metadata as if it's part of the dataset itself in the future, then that property shouldn't be exposed to the developer anyway - they should be accessing it through our functions.

Reply:
I should have read all your comments before answering to the first one, but yes, this is the motivation, but I agree that as long as our API is designed to return types that make sense and work well with each other, it should be okay, we just have to provide the right functions.